src/components/Picker/Picker.tsx

```diff
@@ -77,12 +92,179 @@ class Picker extends React.Component<Props, State> {
       searchItems: [],
       selectedItems: [],
       moreInfo: false,
+      value: '',
+      input: () => { },
+      suggestions: [],
+      chipListState: [],
+      focusArr: [],
+      itemsList:
```

Comment:
Do not add dummy data to component. Instead use data from source props directly. You may remove itemsList also.